src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.cs

```diff
@@ -32,10 +39,18 @@ internal partial class SyntacticClassificationTaggerProvider : ITaggerProvider
         public SyntacticClassificationTaggerProvider(
             IForegroundNotificationService notificationService,
             ClassificationTypeMap typeMap,
+            [Import(AllowDefault = true)] IViewSupportsClassificationService viewSupportsClassificationService,
+            ITextBufferAssociatedViewService associatedViewService,
+            [ImportMany] IEnumerable<Lazy<ILanguageService, LanguageServiceMetadata>> allLanguageServices,
+            [ImportMany] IEnumerable<Lazy<ILanguageService, ContentTypeLanguageMetadata>> contentTypes,
             [ImportMany] IEnumerable<Lazy<IAsynchronousOperationListener, FeatureMetadata>> asyncListeners)
         {
             _notificationService = notificationService;
             _typeMap = typeMap;
+            _viewSupportsClassificationService = viewSupportsClassificationService;
+            _associatedViewService = associatedViewService;
+            _editorClassificationLanguageServices = allLanguageServices.Where(s => s.Metadata.ServiceType == typeof(IEditorClassificationService).AssemblyQualifiedName);
```

Comment:
Note since this is deferred you're going to rerun this all the time. I don't think you care.